ecyp-database.Rmd

```diff
@@ -198,19 +216,23 @@ ecyp_update <- reactive({
     print("Refresh was run")
 
     refresh_table(sheet_url) %>%
-      dplyr::filter(Unique_Number == enter_number())
+      dplyr::filter(`Unique Number` == enter_number()) %>% 
+      t() %>% 
```

Comment:
This spins the table but only when the user will see it. Otherwise its easier to handle the data the other way.